cypress-tests/cypress/e2e/ConnectorUtils/Trustpay.js

```diff
@@ -14,6 +14,58 @@ const successfulThreeDSTestCardDetails = {
     "card_cvc": "737",
 };
 
+const idealBankRedirectDetails = {
+    "bank_redirect": {
+        "ideal": {
+            "billing_details": {
+                "billing_name": "John Doe"
+            },
+            "bank_name": "ing",
+            "preferred_language": "en",
```

Comment:
why did we add preferred language field in all the payment method data? this field is only in sofort pm ryt?